### PR TITLE
update dag config

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 1]
```diff
@@ -124,7 +124,17 @@ There are some differences between installation packages and commands in differe
     VarDataListMaxSize: 100    # If HDFS columns are read, the number is limited to 100 at a time.
   ```
 
-5. Configure the algorithm file path (`exec_file`) in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. If there are multiple machines, ensure that the paths are the same.
+5. Configure the algorithm file path (`exec_file`) only in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. Currently, all `exec_file` parameters are set to the path of the `run_algo.sh` file.
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:127]: the file path is `dag-ctrl/etc/tasks.yaml`.

the file path ---> which file do you specify? as to this sentence, it seems to be the exec_file.

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 2]
```diff
@@ -124,7 +124,17 @@ There are some differences between installation packages and commands in differe
...
+
+  !!! note
+
+        - The algorithm files are provided by NebulaGraph Analytics. Please find directory `scripts` under the installation path of NebulaGraph Analytics above. All algorithm files are in this directory.
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:131]: directory `scripts`  --the directory of `scripts` or the `scripts` directory

ps: do we need to note the path of the `scripts` directory?

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md [hunk 3]
```diff
@@ -124,7 +124,17 @@ There are some differences between installation packages and commands in differe
     VarDataListMaxSize: 100    # If HDFS columns are read, the number is limited to 100 at a time.
   ```
 
-5. Configure the algorithm file path (`exec_file`) in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. If there are multiple machines, ensure that the paths are the same.
+5. Configure the algorithm file path (`exec_file`) only in the file `tasks.yaml`, the file path is `dag-ctrl/etc/tasks.yaml`. Currently, all `exec_file` parameters are set to the path of the `run_algo.sh` file.
+
+  !!! note
+
+        - The algorithm files are provided by NebulaGraph Analytics. Please find directory `scripts` under the installation path of NebulaGraph Analytics above. All algorithm files are in this directory.  
+        - If there are multiple machines, ensure that the paths are the same.
```
Comment (abby-cyber) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:132]: the paths of what?

File: docs-2.0/graph-computing/0.deploy-controller-analytics.md (hunk 1 above)
Comment (abby-cyber) [inline docs-2.0/graph-computing/0.deploy-controller-analytics.md:127]:
```suggestion
5. Configure the algorithm file path (`exec_file`) only in the file `tasks.yaml`, the file path of which is `dag-ctrl/etc/tasks.yaml`. Currently, all `exec_file` parameters are set to the path of the `run_algo.sh` file.
```

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**